api/src/main/scala/uk/ac/wellcome/platform/api/models/DisplayWork.scala

```diff
@@ -6,6 +6,8 @@ import org.elasticsearch.action.get.GetResponse
 import uk.ac.wellcome.models._
 import uk.ac.wellcome.utils.JsonUtil
 
+case class WorksIncludes(identifiers: Boolean = false)
```

Comment:
I think yes to both, if the class grows a companion object then evict it to another file.